### ISSUE TITLE
question about NAN

### ISSUE THREAD

**Issue by yan-yuchen** [alpc91/NICE-GAN-pytorch#27]
When I train the model with learning rate:0.0001,after several echos,I got  "d_loss: nan, g_loss: nan" ,how can I deal with this situation?
Thanks a lot!

**Comment by TMDTom** [comment]
> When I train the model with learning rate:0.0001,after several echos,I got "d_loss: nan, g_loss: nan" ,how can I deal with this situation?
> Thanks a lot!

I meet the same question.I guess maybe the generator is ovefitting or something. You can use the saved checkpoint to evaluate your data.
